hydra-tx/test/Hydra/Tx/Contract/Increment.hs

```diff
@@ -26,14 +25,12 @@ import Hydra.Data.Party qualified as OnChain
 import Hydra.Ledger.Cardano.Time (slotNoFromUTCTime)
 import Hydra.Plutus.Orphans ()
 import Hydra.Tx.ContestationPeriod (ContestationPeriod, toChain)
-import Hydra.Tx.Contract.Deposit (healthyDepositTx, healthyDepositUTxO)
```

Comment:
Made the `Hydra.Tx.Contract.Increment` independent of `Hydra.Tx.Contract.Deposit`, but this is in a way a bit less elegant.